.github/workflows/wasmcloud-control-interface.yml

```diff
@@ -17,6 +17,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+    - name: Update rust
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: stable
+        components: rustfmt, clippy
+    - name: install_protoc
+      run: sudo apt-get install -y protobuf-compiler
```

Comment:
Question, does this mean consumers of this crate now have to install protobuf?

Reply:
I don't know for sure but probably

Reply:
For anyone that finds this .. I believe this is because opentelemetry-otlp (which is enabled with the "otel" flag in wasmbus-rpc) requires protoc as part of its build process. This seems to be required even if you are only using https for otel tracing and not grpc. We should look into the opentelemetry-otlp crate to see if we can submit a PR to make protoc requirement optional